warehouse/snowflake/snowflake_test.go

```diff
@@ -199,19 +177,3 @@ func TestSnowflakeConfigurationValidation(t *testing.T) {
 	testhelper.VerifyingConfigurationTest(t, destination)
 }
 
-func TestMain(m *testing.M) {
-	//_, exists := os.LookupEnv(testhelper.SnowflakeIntegrationTestCredentials)
-	//if !exists {
-	//	log.Println("Skipping Snowflake Test as the Test credentials does not exists.")
-	//	return
-	//}
-	//
-	//handle = &TestHandle{
-	//	WriteKey:             "2eSJyYtqwcFiUILzXv2fcNIrWO7",
-	//	CaseSensiiveWriteKey: "2eSJyYtqwcFYUILzXv2fcNIrWO7",
-	//	Schema:               testhelper.Schema(warehouseutils.SNOWFLAKE, testhelper.SnowflakeIntegrationTestSchema),
-	//	CaseSensitiveSchema:  fmt.Sprintf("%s_%s", testhelper.Schema(warehouseutils.SNOWFLAKE, testhelper.SnowflakeIntegrationTestSchema), "cs"),
-	//	Tables:               []string{"identifies", "users", "tracks", "product_track", "pages", "screens", "aliases", "groups"},
-	//}
```

Comment:
Not sure why those things are needed here

Reply:
Since we have moved to Table Driven Tests these are not required here.

Reply:
Even the global handler is not required.